crates/agglayer-storage/src/stores/state/tests.rs

```diff
@@ -135,3 +137,95 @@ fn can_detect_inconsistent_state(network_id: NetworkId, store: StateStore) {
         Err(Error::InconsistentState { .. })
     ));
 }
+
+use pessimistic_proof_test_suite::sample_data::{self as data};
+
+#[rstest]
+fn can_read(network_id: NetworkId, store: StateStore) {
+    let certificates: Vec<Certificate> = [
+        "n15-cert_h0.json",
+        "n15-cert_h1.json",
+        "n15-cert_h2-v2.json",
+        //"n15-cert_h3-v2.json",
```

Comment:
Should it be uncommented or is it a leftover?

Reply:
Leftover that I removed, this certificate was an attempt to replicate the h3 issue, I also renamed `n15-cert_h2-v2.json` to `n15-cert_h2.json` and deleted the unused ones